systemtests/src/test/java/io/enmasse/systemtest/sharedinfra/MessagingInfraTest.java

```diff
@@ -10,41 +10,39 @@
 import io.enmasse.systemtest.TestTag;
 import io.enmasse.systemtest.bases.TestBase;
 import io.enmasse.systemtest.bases.isolated.ITestIsolatedSharedInfra;
+import io.enmasse.systemtest.messaginginfra.crds.MessagingInfraCrd;
 import io.enmasse.systemtest.time.TimeoutBudget;
 import io.enmasse.systemtest.utils.TestUtils;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import java.time.Duration;
 import java.util.Collections;
-import java.util.List;
 import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 @Tag(TestTag.ISOLATED_SHARED_INFRA)
 public class MessagingInfraTest extends TestBase implements ITestIsolatedSharedInfra {
+
     @Test
     public void testInfraStaticScalingStrategy() throws Exception {
-        MessagingInfra infra = new MessagingInfraBuilder()
-                .editOrNewMetadata()
-                .withName("infra1")
+        MessagingInfra infra = MessagingInfraCrd.getDefaultInfra()
```

Comment:
I think for readability its better to use builder and specifically set metadata rather than using getDefaultInfra(). getDefaultInfra() has its uses, but its when you want defaults and not when you want a specific infra IMO.